<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="BOMC9082093.local" ignored="1" name="Test" tests="1" failures="1" timestamp="2021-03-04T22:17:43 COT" time="2.078" errors="0">
  <testcase name="@AfterMethod afterMethod" time="2.004" classname="com.com.bancodebogota.test.Prueba1">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.com.bancodebogota.test.Prueba1.afterMethod(Prueba1.java:89)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- @AfterMethod afterMethod -->
  <testcase name="f1" time="0.004" classname="com.com.bancodebogota.test.Prueba1">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.com.bancodebogota.test.Prueba1.f1(Prueba1.java:43)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- f1 -->
  <testcase name="f2" time="0.0" classname="com.com.bancodebogota.test.Prueba1">
    <ignored/>
  </testcase> <!-- f2 -->
</testsuite> <!-- Test -->
